pkg/models/plan_general_characteristics.go

```diff
@@ -36,6 +36,8 @@ type PlanGeneralCharacteristics struct {
 	HasComponentsOrTracksNote   *string `json:"hasComponentsOrTracksNote" db:"has_components_or_tracks_note"`
 
 	// Page 2
+	PhasedIn                     *bool          `json:"phasedIn" db:"phased_in" statusWeight:"1"` //default to false
```

Comment:
How is this defaulted to false? Or is this just saying FE will default to false?